constellations/orion/src/domains/regex.go

```diff
@@ -7,6 +7,9 @@ const (
 	// Ensures at least one uppercase or lowercase letter
 	REGEX_LETTER = `[A-Za-z]`
 
+	// School names (alphabetic and some symbols)
+	REGEX_SCHOOLS = `[^a-zA-Z\s.-]`
```

Comment:
hermm REGEX_LETTER_BASIC_SYMBOLS (i think).
There will most likely be another thing that will need this. So if there's ever a time we want to differentiate something school specific, we can rename to REGEX_SCHOOL. But for now, let's use something general.